include/lmms_basics.h

```diff
@@ -141,6 +141,12 @@ constexpr const char* UI_CTRL_KEY =
 #else
 "Ctrl";
 #endif
+constexpr const char* UI_ALT_KEY =
```

Comment:
Just so you note, I'm planning to remove this entire file eventually, so would be good if we find a good place in case i go through it.

Reply:
Thanks for the heads up, however I would expect this to be handled as part of that removal (i.e. "refactor").